Italian/caramelle.md

```diff
@@ -18,3 +18,5 @@ Che fai?
 [Spuntini] (spuntini/spuntini.md)
 
 [Alice non abita più qui] (alice/alice.md)
+
+[Vado in bagno](bagno.bagno.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.